include/hip/spirv_hip_devicelib.hh

```diff
@@ -529,8 +493,8 @@ EXPORT float __saturatef(float x) {
 }
 
 EXPORT void __sincosf(float x, float *sptr, float *cptr) {
-  *sptr = GEN_NAME2(sin_native, f)(x);
-  *cptr = GEN_NAME2(cos_native, f)(x);
+  *sptr = sin(x);
+  *cptr = cos(x);
```

Comment:
So what is the difference between `::sin` and `sin`?

Reply:
`::xyz` will look up for symbol in the root namespace only. `xyz` will look up for the symbol in the current namespaces or in the namespaces declared in `using <namespace>` statements.